cypress/support/step_definitions/manualAccountCreation/manualAccountCreation.ts

```diff
@@ -294,18 +294,18 @@ When('I enter mobile telephone number {string}', (mobileTelephone: string) => {
 When('I enter home telephone number {string}', (homeTelephone: string) => {
   contactDetails.enterHomeTelephone(homeTelephone);
 });
-When('I enter business telephone number {string}', (businessTelephone: string) => {
-  contactDetails.enterBusinessTelephone(businessTelephone);
+When('I enter work telephone number {string}', (workTelephone: string) => {
+  contactDetails.enterWorkTelephone(workTelephone);
 });
 
 Then(
   'I verify {string},{string},{string},{string},{string} on contact details page',
-  (primaryEmail, secondaryEmail, mobileTelephone, homeTelephone, businessTelephone: string) => {
+  (primaryEmail, secondaryEmail, mobileTelephone, homeTelephone, workTelephone: string) => {
     cy.get('#primaryEmailAddress').should('have.value', primaryEmail);
     cy.get('#secondaryEmailAddress').should('have.value', secondaryEmail);
     cy.get('#mobileTelephoneNumber').should('have.value', mobileTelephone);
     cy.get('#homeTelephoneNumber').should('have.value', homeTelephone);
-    cy.get('#businessTelephoneNumber').should('have.value', businessTelephone);
+    cy.get('#workTelephoneNumber').should('have.value', workTelephone);
   },
 );
```

Comment:
POM shouldnt be used, and generic steps can be used instead here

Reply:
I haven't refactor framework. the ticket still there in backlog. It would be great If someone from test team will pickup  that ticket

Reply:
That ticket hasn't yet been completed, but we shouldn't be actively creating new steps that we know will need to be removed and refactored. Going forward we need to use the new approach and not fall back to steps that will be removed.